### ISSUE TITLE
Listboxes are not reading Labels in IE 11

### ISSUE THREAD

**Issue by HerinHentry** [nvaccess/nvda#7082]
Tested in Windows 7 / IE 11/ NVDA 2016.3

Sample page:
https://jsfiddle.net/mightyyouth/3u78u9q8/

Code:
```
<body>	
<Label for="idphcode"> Select a Phone </Label>		
<select size="3" name="phcode" id="idphcode" style="width: 100%">	
    <option value="1001">Nokia</option>
	  <option value="1002">HP</option>
	  <option value="1003">iPhone</option>				
</select>
</body>
```

Issue:
Labels even though are associated, Chrome and Firefox with NVDA reads the Label. IE 11 does not read the Label. 

Note:
Aria-Labelledby works + Combo boxes works
<Select> tag with size and multiple attributes does not work.

Thanks,
Herin

**Comment by feerrenrut** [comment]
Steps to reproduce, compare the output of chrome and IE when doing the following:
* Go to https://jsfiddle.net/mightyyouth/3u78u9q8/embedded/result/#Result
* Tab through the window.

**Comment by stickbear2015** [comment]
I can't reproduce this under IE 11 or chrome, using NVDA 2017.1 and windows 10. From your initial opening of this ticket, your using 2016.3, please upgrade to 2017.1 and report if the issue still persists.

**Comment by HerinHentry** [comment]
I have upgrade to NVDA 2017.1 and issue still persists.

Output below:
https://fiddle.jshell.net/mightyyouth/3u78u9q8/show/light/  frame  MSAAHTML Registered Handler
by mightyyouth  document
list
Nokia  not selected

As I use down arrow,
Nokia
HP
iPhone

**Comment by LadynRed** [comment]
I have updated to 2017.2, Windows 7, Firefox 53.0.3 - have a select box for a list of states, and NVDA does not read any of the option fields. It will announce 'list item x of 20', but it will not read the value.  Forms mode, using up and down arrow keys. Even tried the Alt+down arrow, doesn't matter.  I don't want to twist our developers in knots if this is an NVDA issue.

**Comment by jcsteh** [comment]
[**@LadynRed**](https://github.com/LadynRed) commented on [13 Jun 2017, 05:21 GMT+10](https://github.com/nvaccess/nvda/issues/7082#issuecomment-307892856 "2017-06-12T19:21:35Z - Replied by Github Reply Comments"):
> Not IE, but similar issue - I have updated to 2017.2, Windows 7, Firefox 53.0.3 - have a select box for a list of states, and NVDA does not read any of the option fields.

Please file a new issue including full details (as requested by the template) and a test case which demonstrates the issue. Thanks.

**Comment by feerrenrut** [comment]
I think some investigation needs to be done to determine if this is a browser specific issue. That said IE 11 is fairly low priority for us. I will set the priority 3 for this investigation. If anyone wants to do this investigation and report back, that would be fantastic.

**Comment by Adriani90** [comment]
I can still reproduce this issue as follows in NVDA 2019.1 and IE11:
1. Copy the code from the description into an empty file
2. Save it as html
3. Open it in IE
4. Press nvda+spacebar to change to focus mode
5. Press tab and shift+tab

Actual: NVDA reports "list Nokia"
Expected: NVDA should report "Select a Fone: List Nokia"

**Comment by Adriani90** [comment]
IE support ended completely in 2022, and the browser is automatically removed from machines that still have it after an Edge update in February 2023. I am closing this as abandoned, since I don't expect any further accessibility updates to IE or to IE mode in Edge. If you have still this issue in other browsers, please open a new issue with updated steps to reproduce and a minimal test case.
